src/app/shared/services/mailing.js

```diff
@@ -0,0 +1,32 @@
+import * as emailjs from 'emailjs-com';
+
+export default class Mailing {
+    constructor($mdToast, $translate) {
+        this.$mdToast = $mdToast;
+        this.$translate = $translate;
+        emailjs.init(process.env.EMAILJS_API_KEY);
+    }
+
+    contact(answer) {
+        emailjs.send(process.env.EMAILJS_SERVICE, process.env.EMAILJS_TEMPLATE, answer)
+            .then((response) => {
+                this.$mdToast.show(
+                    this.$mdToast.simple()
+                    .textContent(this.$translate.instant('TOAST.CONTACT.SUCCESS'))
+                    .position('bottom right')
+                    .theme('success-toast')
+                    .hideDelay(3000)
+                );
+            }).catch((error) => {
+                this.$mdToast.show(
+                    this.$mdToast.simple()
+                    .textContent(this.$translate.instant('TOAST.CONTACT.ERROR'))
+                    .position('bottom right')
+                    .theme('error-toast')
+                    .hideDelay(3000)
```

Comment:
The code of this two toast are really similar, you can create a function with 'message' and 'toastTime' as dependencies